src/components/forms/Select/index.js

```diff
@@ -121,9 +122,17 @@ module.exports = React.createClass({
   },
 
   renderOptions: function(bool) {
+    if (!bool) {
+      return false;
+    }
+    var options = this.getFilteredOptions();
+    var key = this.props.labelKey;
+    var value = this.state.value;
+    var fauxFocus = value ? _.findIndex(options, function(obj) { return obj[key] == value; }) : -1;
```

Comment:
fauxFocus = forEach(function(d, i) { return d[key] === value && i }) || 0; // something like this. Basically pass the component what it needs, don't clean up in child components.